docs/verifiers/ceph-osd.rst

```diff
@@ -19,7 +19,7 @@ are part of are healthy (Note: if more than one Ceph cluster exists, verified
 ``ceph-osd`` units may be part of more than one cluster). A cluster is
 considered healthy if the action's output contains HEALTH_OK.
 
-.. image:: check_ceph_cluster_health.svg
+.. image:: ../img/check_ceph_cluster_health.svg
```

Comment:
`tox -e docs` fails on my laptop because:

```
Warning, treated as error:
/tmp/juju-verify/docs/verifiers/ceph-osd.rst::image file not readable: img/check_ceph_cluster_health.svg
```

in your branch, I only see an image in `docs/img/`. Have you moved the images from the verifiers folder or have they been removed?

Reply:
I didn't find a better solution and I use a symbolic link, I just forgot to add it to the confirmation.
```bash
$ ll img
lrwxrwxrwx 1 rg rg 9 Jul 28 12:03 img -> docs/img/
```